src/chemistry/chemTabModel.cpp

```diff
@@ -0,0 +1,277 @@
+#include "chemTabModel.hpp"
```

Comment:
The chemTabModel was moved and refactored into src/eos/chemTab.hpp with api changes.  You need to move any required changes into this class and delete this one.